<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Livreur</class>
 <widget class="QDialog" name="Livreur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>821</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QGroupBox {
 
 background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, radius: 1.35, stop: 0 #fff, stop: 1 #888) ;/*background: linear-gradient(rgba(0,0,0,.5), rgba(0,0,0,.5)),url(feuille.jpg); 
    background-position: center;
    background-size: cover;
    display: flex;
    justify-content: center;
    align-items: center;
    font-family: 'Poppins', sans-serif*/
 
 color: rgb(255, 255, 255);
  border: 1px solid #ffffff;

    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 10px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF);
}


QPushButton {
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background-color:blue
}

 QPushButton:hover:!pressed{ background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF); } 
QPushButton:hover { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF); }</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Home">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">QGroupBox {
	background-image: url(:/img/img/livreur.png);
 
 background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, radius: 1.35, stop: 0 #fff, stop: 1 #888) ;/*background: linear-gradient(rgba(0,0,0,.5), rgba(0,0,0,.5)),url(feuille.jpg); 
    background-position: center;
    background-size: cover;
    display: flex;
    justify-content: center;
    align-items: center;
    font-family: 'Poppins', sans-serif*/
 
 color: rgb(255, 255, 255);
  border: 1px solid #ffffff;

    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 10px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF);
}
</string>
       </property>
       <property name="title">
        <string>Smart Printing System</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>431</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bienvenue, vous êtes le livreur !!</string>
        </property>
       </widget>
       <widget class="QPushButton" name="livrButton">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>190</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Livraisons</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deconnecterButton">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>370</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#evilButton { background-color: red };</string>
        </property>
        <property name="text">
         <string>Déconnecter</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="livraisons">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Smart Printing System</string>
       </property>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>771</width>
          <height>431</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Ajouter</string>
         </attribute>
         <widget class="QPushButton" name="homeButton_4">
          <property name="geometry">
           <rect>
            <x>650</x>
            <y>360</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 127);
</string>
          </property>
          <property name="text">
           <string>Home</string>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter_3">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>20</y>
            <width>321</width>
            <height>361</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 170, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Ajouter une livraison :</string>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>id</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_id"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>date</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="la_date"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>frais</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="le_frais"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>localisation</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="l_adress"/>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="ajouter">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </widget>
         <widget class="QTextBrowser" name="textBrowser_2">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>310</y>
            <width>121</width>
            <height>111</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/img/Projet qt/delivery.png);</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="textBrowser_5">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>-20</y>
            <width>781</width>
            <height>431</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/img/img/j.png);</string>
          </property>
         </widget>
         <zorder>textBrowser_5</zorder>
         <zorder>homeButton_4</zorder>
         <zorder>splitter_3</zorder>
         <zorder>textBrowser_2</zorder>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Afficher</string>
         </attribute>
         <widget class="QTableView" name="table_livraison">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>651</width>
            <height>211</height>
           </rect>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter_2">
          <property name="geometry">
           <rect>
            <x>490</x>
            <y>230</y>
            <width>141</width>
            <height>161</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPushButton" name="pushButton_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_11">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 170, 255);</string>
           </property>
           <property name="text">
            <string>rechercher</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_13">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);</string>
           </property>
           <property name="text">
            <string>Trier</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>220</y>
            <width>311</width>
            <height>199</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>id</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>frais</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="le_id_2"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="le_frais_2"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="la_date_2"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>date</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>localisation</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="l_adress_2"/>
           </item>
          </layout>
         </widget>
         <widget class="QTextBrowser" name="textBrowser_3">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>310</y>
            <width>121</width>
            <height>111</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/img/Projet qt/delivery.png);</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="textBrowser_6">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>-30</y>
            <width>731</width>
            <height>231</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/img/img/j.png);</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="textBrowser_4">
          <property name="geometry">
           <rect>
            <x>-400</x>
            <y>10</y>
            <width>551</width>
            <height>211</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/img/img/j.png);</string>
          </property>
         </widget>
         <zorder>textBrowser_4</zorder>
         <zorder>textBrowser_6</zorder>
         <zorder>layoutWidget</zorder>
         <zorder>table_livraison</zorder>
         <zorder>splitter_2</zorder>
         <zorder>textBrowser_3</zorder>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>supprimer</string>
         </attribute>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>310</y>
            <width>121</width>
            <height>111</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/img/Projet qt/delivery.png);</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_id_sup">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>150</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_21">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>110</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 0, 255);

font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Id livraison :</string>
          </property>
         </widget>
         <widget class="QPushButton" name="supprimer">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>200</y>
            <width>151</width>
            <height>51</height>
           </rect>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>supprimer</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="textBrowser_7">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>271</width>
            <height>192</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/img/img/BBBB.png);</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="textBrowser_8">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>190</y>
            <width>291</width>
            <height>211</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/img/img/PPPPP.png);</string>
          </property>
         </widget>
         <zorder>le_id_sup</zorder>
         <zorder>label_21</zorder>
         <zorder>supprimer</zorder>
         <zorder>textBrowser</zorder>
         <zorder>textBrowser_7</zorder>
         <zorder>textBrowser_8</zorder>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
